themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties

```diff
@@ -0,0 +1,314 @@
+recaptchaNotConfigured=Recaptcha jest wymagana, ale nie skonfigurowana
+consentDenied=Zgoda odrzucona.
+
+noAccount=Nie masz konta?
+username=Nazwa użytkownika (login)
+usernameOrEmail=Nazwa użytkownika lub e-mail (login)
+firstName=Imię
+givenName=Nadane imię
+fullName=Pełne imię i nazwisko
+lastName=Nazwisko
+familyName=Nazwisko rodowe
+email=E-mail
+password=Hasło
+passwordConfirm=Potwierdź hasło
+passwordNew=Nowe hasło
+passwordNewConfirm=Potwierdzenie nowego hasła
+rememberMe=Zapamiętaj mnie
+authenticatorCode=Kod jednorazowy
+address=Adres
+street=Ulica
+locality=Miejscowość
+region=Województwo
+postal_code=Kod pocztowy
+country=Państwo
+emailVerified=Email zweryfikowany
+gssDelegationCredential=Świadectwo przekazania uprawnień GSS
+
+profileScopeConsentText=Profil użytkownika
+emailScopeConsentText=Adres email
+addressScopeConsentText=Adres
+phoneScopeConsentText=Numer telefonu
+offlineAccessScopeConsentText=Dostęp offline
+samlRoleListScopeConsentText=Moje role
+
+loginTotpIntro=Aby uzyskać dostęp do tego konta, musisz skonfigurować generator haseł jednorazowych
+loginTotpStep1=Zainstaluj jedną z następujących aplikacji na telefonie komórkowym
+loginTotpStep2=Otwórz aplikację i zeskanuj kod kreskowy
+loginTotpStep3=Wprowadź jednorazowy kod podany przez aplikację i kliknij Prześlij aby zakończyć konfigurację
+loginTotpManualStep2=Otwórz aplikację i wprowadź klucz
+loginTotpManualStep3=Użyj poniższych wartości konfiguracji, jeśli aplikacja pozwala na ich ustawienie
+loginTotpUnableToScan=Nie można skanować?
+loginTotpScanBarcode=Zeskanować kod paskowy?
+loginTotpOneTime=Kod jednorazowy
+loginTotpType=Typ
+loginTotpAlgorithm=Algorytm
+loginTotpDigits=Cyfry
+loginTotpInterval=Interwał
+loginTotpCounter=Licznik
+
+loginTotp.totp=Oparte o czas
+loginTotp.hotp=Oparte o licznik
+
+
+oauthGrantRequest=Czy przyznajesz te uprawnienia dostępu?
+inResource=w
+
+emailVerifyInstruction1=Została wysłana do Ciebie wiadomość e-mail z instrukcjami jak zweryfikować swój adres e-mail.
+emailVerifyInstruction2=Nie otrzymałem kod weryfikacyjny w wiadomości e-mail?
+emailVerifyInstruction3=aby ponownie wysłać wiadomość e-mail.
+
+emailLinkIdpTitle=Link {0}
+emailLinkIdp1=Wiadomość e-mail z instrukcjami, aby powiązać konto {0} {1} z kontem {2} została wysłana do Ciebie.
+emailLinkIdp2=Nie otrzymałem kod weryfikacyjny w wiadomości e-mail?
+emailLinkIdp3=aby ponownie wysłać wiadomość e-mail.
+emailLinkIdp4=Jeśli już zweryfikowana e-mail w innej przeglądarce
+emailLinkIdp5=aby kontynuować.
+
+backToLogin=&laquo; Powrót do logowania
+
+emailInstruction=Wpisz swój adres e-mail lub nazwę użytkownika a wyślemy do Ciebie instrukcje, jak utworzyć nowe hasło.
+
+copyCodeInstruction=Proszę skopiować ten kod i wklej go do aplikacji\:
+
+pageExpiredTitle=Strona wygasła
+pageExpiredMsg1=Aby ponownie uruchomić proces logowania
+pageExpiredMsg2=Aby kontynuować proces logowania
+
+personalInfo=Informacje osobiste\:
+role_admin=Admin
+role_realm-admin=Królestwo Admin
```

Comment:
This one is pretty tricky (and funny) at the same time. 

A term `realm` can be translated to `królestwo`, which is more like a `kingdom` in English :) Looking into the dictionary, none o the terms fits here to be honest. But if I had to choose, I would pick `strefa`, which can be translated into `zone`. Even though it's still wrong, it's slightly better than the `królestwo` I think :)